cloudprem/app/charts/kubed/.helmignore

```diff
@@ -14,10 +14,10 @@
 *.swp
 *.bak
 *.tmp
-*.orig
 *~
 # Various IDEs
 .project
 .idea/
 *.tmproj
-.vscode/
```

Comment:
Why is this being removed? Same for *.orig on L17.

Reply:
This appears to be GitHub getting confused. It thinks that the .helmignore from the frontegg integrations-service chart was replaced by the helmignore for the kubed chart. I'm not sure exactly what's goin on here but these files are maintained by the chart maintainers and are not like gitignore files. It basically tells helm to ignore certain files when processing the templates. With the frontegg chart I had to do some creative terraform to insert a helm ignore file at runtime with terraform but with the kubed chart I didn't have to so this is the original file.